tests/lib/rules/await-async-queries.test.ts

```diff
@@ -261,6 +261,24 @@ ruleTester.run(RULE_NAME, rule, {
       `
 		),
 
+		// jasmine async matchers are valid
```

Comment:
I'm trying to think any invalid case that should be added, but there is none probably, since the improvement only includes more valid combinations.